### PR TITLE
Query Expansion function 

### PR DESCRIPTION
**Pull request by SalsabilTasnia**

### CODE REVIEW

File: search.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 from collections import Counter
 from index import Posting, PostingList
 from encode import decode
-
+from nltk.corpus import wordnet
```
Comment (hcwong) [inline search.py:13]: 👍 . nice TIL wordnet dont even need to make http request anymore

Comment (SalsabilTasnia) [reply search.py:13]: Haha yep it doesnt. Its part of nltk so it can be used like a library

Comment (SalsabilTasnia) [reply search.py:13]: Is the pr ok for merging?

Comment (hcwong) [reply search.py:13]: Go ahead

**Review by hcwong — APPROVED** [review]

File: search.py (hunk 1 above)
Comment (SalsabilTasnia) [reply search.py:13]: ok merged!!

**Pull request merged**